tests/integration.rs

```diff
@@ -81,6 +82,30 @@ pub fn netflix()
         .unwrap();
 }
 
+#[cfg(feature = "trust_anchor_util")]
+#[test]
+pub fn ed25519()
+{
+    let ee = include_bytes!("ed25519/ee.der");
+    let ca = include_bytes!("ed25519/ca.der");
+
+    let ee_input = untrusted::Input::from(ee);
+    let inter_vec = vec![];
```

Comment:
I think `inter_vec` should just be `let intermediates = &[];`

Reply:
Hopefully addressed, left out a variable for intermediates at all and passing &[] directly into verify_is_valid_tls_server_cert().